schedule/application_test.go

```diff
@@ -37,7 +41,7 @@ func (s *ApplicationTestSuite) TestCallAndCommand() {
 	delayIfStillRunningCall := 0
 	skipIfStillRunningCall := 0
 
-	app := NewApplication(mockArtisan, nil, mockLog)
+	app := NewApplication(mockArtisan, nil, mockLog, mockConfig.GetBool("app.debug"))
```

Comment:
S: Do we really need to use `mockConfig`? We can pass `false` directly since we are not testing the config.